unstructured/ingest/v2/logger.py

```diff
@@ -0,0 +1,111 @@
+import ast
+import json
+import os
+from logging import Formatter, Logger, StreamHandler, getLevelName, getLogger
+from typing import Any, Callable
+
+log_level = os.getenv("INGEST_LOG_LEVEL", "INFO")
+LOGGER_NAME = "unstructured.ingest.v2"
+
+
+def default_is_data_sensitive(k: str, v: Any) -> bool:
```

Comment:
Are we still able to use ` enhanced_field(sensitive=True)` ?  I really liked how that worked.

Reply:
This was mostly a copy-paste of what was already being used by the current ingest code.

Reply:
To be explicit, only the logging code was copied. Everything else in this PR is new code.